colabs/artifact_basics/Artifact_Basics.ipynb

```diff
@@ -0,0 +1,351 @@
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "q2EeMdcpC7Dl"
+      },
+      "source": [
+        "In order to log data to our web service, you need to log in and import `wandb`. If this is your first time using W&B, you'll need to sign up for a free account at the link that appears."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "nwEF5d7NChdB"
+      },
+      "outputs": [],
+      "source": [
+        "!pip install wandb\n",
+        "import wandb\n",
+        "wandb.login()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "vp9Zhzp5Hvt1"
+      },
+      "source": [
+        "# Create An Artifact"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3xEsZzOEnPFY"
+      },
+      "source": [
+        "The general workflow for creating an Artifact is:\n",
+        "\n",
+        "\n",
+        "1.   Intialize a run.\n",
+        "2.   Create an Artifact.\n",
+        "3.   Add a dataset, a model, or another Artifact as an input.\n",
+        "4.   Log the artifact in the W&B platform.\n",
+        "\n",
+        "This can by accomplished in two lines of code:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Vndk289hgVVK"
+      },
+      "outputs": [],
+      "source": [
+        "run = wandb.init(project=\"artifact-basics\")\n",
+        "run.log_artifact(artifact_or_path=\"/content/sample_data/mnist_test.csv\", name=\"my_first_artifact\", type=\"dataset\")\n",
+        "run.finish()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "tgzxAogXnwCj"
+      },
+      "source": [
+        "First, initalize the run with `wandb.init()`. In this demo, the code adds the run to the `artifact-basic` project, but you can change the name to anyting you'd like.\n",
+        "\n",
+        "Next, log the artifact with `wandb.artifact()`. In this demo, the artifact is a `dataset` using data from `mnist_test.csv`. You can customize your artifact with a `name` and other metadata- see the Artifacts Reference guide for more information.\n",
+        "\n",
+        "\n",
+        "If you change or add any parameters names, be sure to replicate those changes in the following code samples as well."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "bpGidixAICM1"
+      },
+      "source": [
+        "# Use an Artifact"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "jZC69sSMdX4j"
+      },
+      "source": [
+        "When you need to select a particular Artifact, indicate the artifact you want to use with the `use_artifact` method. This returns a run object. In the proceeding code snippet specifies an artifact called `my_first_artifact` with the alias `latest`:\n"
```

Comment:
The language around select is not very clear - what does it mean to programmatically select an artifact? Can we be more specific here? To pull down? To reference? To use it in a run?

Reply:
I can't actually find this line anymore, so I'm just going to resolve this.